src/drive/users/types.ts

```diff
@@ -28,12 +28,13 @@ export type UpdateProfilePayload = Partial<Pick<UserSettings, 'name' | 'lastname
 
 export type PreCreateUserResponse = {
   publicKey: string;
+  publicKyberKey: string;
```

Comment:
Maybe we can make this parameter optional for now. Wdyt? @sg-gs

Reply:
Yes, as long as the backend is not offering it for sure we need to reflect the reality of the types